app/assets/stylesheets/components/_chart.scss

```diff
@@ -5,6 +5,10 @@
   margin-bottom: 0;
 }
 
+.app-c-chart__table-wrapper {
+  overflow: auto;
+}
+
 .app-c-chart__table {
   overflow: auto;
```

Comment:
Can this be removed now? It looks to me that it might be redundant with this change.

Reply:
yes, my mistake caused by breaking my branch and starting again. Now fixed.

Reply:
:+1: